src/pages/Home/index.tsx

```diff
@@ -55,6 +55,14 @@ function Home() {
   return (
     <main>
       {loading && <CircularProgress />}
+      {!loading && (
+        <Typography variant='h5'>
+          The top CS conferences are listed in{' '}
+          <Link variant='h5' target='_blank' href={`http://CSRankings.org`}>
```

Comment:
All links should point to https (if supported by the destination)
https://csrankings.org/